app_dart/test/request_handlers/github_webhook_test.dart

```diff
@@ -1077,7 +1077,14 @@ void main() {
             '"github_link","value":'
             '"https://github.com/flutter/cocoon/pull/583"}],"notify":'
             '{"pubsubTopic":"projects/flutter-dashboard/topics/luci-builds",'
-            '"userData":"eyJyZXRyaWVzIjowfQ=="}}}]}]');
+            '"userData":"eyJyZXRyaWVzIjowfQ=="}}}]},'
+            '{"requests":[{"searchBuilds":{"predicate":{"builder":{"project":'
+            '"flutter","bucket":"try"},"createdBy":"test@test","tags":[{"key":'
+            '"buildset","value":"pr/git/583"},{"key":"github_link","value":'
+            '"https://github.com/flutter/cocoon/pull/583"},{"key":"user_agent",'
+            '"value":"flutter-cocoon"}]}}},{"searchBuilds":{"predicate":{"builder":'
+            '{"project":"flutter","bucket":"try"},"tags":[{"key":"buildset","value":'
+            '"pr/git/583"},{"key":"user_agent","value":"recipe"}]}}}]}]');
       }
```

Comment:
This JSON is too long and complicated, can we make it more readable? Same to the below literals.

Reply:
Done